modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js

```diff
@@ -154,6 +155,11 @@ class Sidebar extends Component {
 	disposeInternal() {
 		super.disposeInternal();
 
+		document.removeEventListener(
+			'mousedown',
+			this._handleDocumentMouseDown
```

Comment:
Not sure, but this one _might_ not work either. You passed `false` in the subscription, but not in the removal. [MDN says](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#Matching_event_listeners_for_removal):

> While `addEventListener()` will let you add the same listener more than once for the same type if the options are different, the only option `removeEventListener()` checks is the `capture`/`useCapture` flag. Its value must match for removeEventListener() to match, but the other values don't.

If I read that correctly, it probably works, but I am not 100% sure. Maybe it is better to pass matching flags just in case, especially because of:

> It's worth noting that some browser releases have been inconsistent on this, and unless you have specific reasons otherwise, it's probably wise to use the same values used for the call to `addEventListener()` when calling `removeEventListener()`.